### ISSUE TITLE
model file not found

### ISSUE THREAD

**Issue by vcvishal** [kozistr/Awesome-Segmentations#2]
where these model files
Implementation List
FCNet
SegNet
U-Net
FusionNet
FC-DenseNet
ENet
LinkNet
RefineNet
PSPNet
Mask R-CNN
DecoupledNet
GAN-SS
G-FRNet

thank you

**Comment by kozistr** [comment]
previously. i have already done writing the codes  while playing the kaggle challenges,
but, something just was wrong, and most of my implementation was gone. uncommitted : (

sorry for that : (

if you need any segmentation model (scripts) files w/ pre-trained models in *keras*, you can check out this [repo](https://github.com/qubvel/segmentation_models)

thanks in advance
